logic.js

```diff
@@ -0,0 +1,59 @@
+const Tweeter = function () {
+    let posts = [ //our posts array
+        // {
+        //     text: "First post!",
+        //     id: "p1",
+        //     comments: [
+        //         { id: "c1", text: "First comment on first post!" },
+        //         { id: "c2", text: "Second comment on first post!!" },
+        //         { id: "c3", text: "Third comment on first post!!!" }
+        //     ]
+        // },
+        // {
+        //     text: "Aw man, I wanted to be first",
+        //     id: "p2",
+        //     comments: [
+        //         { id: "c4", text: "Don't wory second poster, you'll be first one day." },
+        //         { id: "c5", text: "Yeah, believe in yourself!" },
+        //         { id: "c6", text: "Haha second place what a joke." }
+        //     ]
+        // }
+    ]
+    let postIdCounter = 1 //count our post for id
+    let commentIdCounter = 1 //count our comments for id
+    const getPosts = () => posts //returns our posts array
```

Comment:
Leave an empty line between the different functions. We need to let our code breathe a little.